src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-automation-module/main.combined.bicep

```diff
@@ -0,0 +1,12 @@
+resource automationAccount_automationVariable 'Microsoft.Automation/automationAccounts/modules@2015-10-31' = {

```

Comment:
TODO: I'll update this to use parent-child syntax
https://github.com/Azure/bicep/blob/main/docs/spec/resource-scopes.md#parent-child-syntax

Reply:
Updated here:
https://github.com/Azure/bicep/commit/ac634afda59d439b448f45ffdc33c5d25f592fad